.vscode/tasks.json

```diff
@@ -19,7 +43,7 @@
             "type": "process",
             "args": [
                 "publish",
-                "${workspaceFolder}/src/OctoshiftCLI/OctoshiftCLI.csproj",
+                "${workspaceFolder}/src/gei/gei.csproj",
```

Comment:
can we either change this to publish the sln instead of the csproj (that will build everything), add 2x publish tasks for each CLI, or possibly just remove the publish task entirely, I'm not sure when somebody would ever need that.

Reply:
Removed.